--- qmmp-0.2.3/src/ui/forms/aboutdialog.ui~	2008-07-22 13:10:39.000000000 +0200
+++ qmmp-0.2.3/src/ui/forms/aboutdialog.ui	2009-01-08 16:10:44.000000000 +0100
@@ -16,18 +16,6 @@
    <property name="spacing" >
     <number>6</number>
    </property>
-   <property name="leftMargin" >
-    <number>9</number>
-   </property>
-   <property name="topMargin" >
-    <number>9</number>
-   </property>
-   <property name="rightMargin" >
-    <number>9</number>
-   </property>
-   <property name="bottomMargin" >
-    <number>9</number>
-   </property>
    <item>
     <widget class="QLabel" name="pixmapLabel" >
      <property name="text" >
@@ -51,24 +39,6 @@
        <string>About</string>
       </attribute>
       <layout class="QGridLayout" >
-       <property name="leftMargin" >
-        <number>9</number>
-       </property>
-       <property name="topMargin" >
-        <number>9</number>
-       </property>
-       <property name="rightMargin" >
-        <number>9</number>
-       </property>
-       <property name="bottomMargin" >
-        <number>9</number>
-       </property>
-       <property name="horizontalSpacing" >
-        <number>6</number>
-       </property>
-       <property name="verticalSpacing" >
-        <number>6</number>
-       </property>
        <item row="0" column="0" >
         <widget class="QTextEdit" name="aboutTextEdit" >
          <property name="readOnly" >
@@ -83,24 +53,6 @@
        <string>Authors</string>
       </attribute>
       <layout class="QGridLayout" >
-       <property name="leftMargin" >
-        <number>9</number>
-       </property>
-       <property name="topMargin" >
-        <number>9</number>
-       </property>
-       <property name="rightMargin" >
-        <number>9</number>
-       </property>
-       <property name="bottomMargin" >
-        <number>9</number>
-       </property>
-       <property name="horizontalSpacing" >
-        <number>6</number>
-       </property>
-       <property name="verticalSpacing" >
-        <number>6</number>
-       </property>
        <item row="0" column="0" >
         <widget class="QTextEdit" name="authorsTextEdit" >
          <property name="readOnly" >
@@ -132,18 +84,6 @@
        <property name="spacing" >
         <number>6</number>
        </property>
-       <property name="leftMargin" >
-        <number>9</number>
-       </property>
-       <property name="topMargin" >
-        <number>9</number>
-       </property>
-       <property name="rightMargin" >
-        <number>9</number>
-       </property>
-       <property name="bottomMargin" >
-        <number>9</number>
-       </property>
        <item>
         <widget class="QTextEdit" name="thanksToTextEdit" >
          <property name="readOnly" >
@@ -158,24 +98,6 @@
        <string>License Agreement</string>
       </attribute>
       <layout class="QGridLayout" >
-       <property name="leftMargin" >
-        <number>9</number>
-       </property>
-       <property name="topMargin" >
-        <number>9</number>
-       </property>
-       <property name="rightMargin" >
-        <number>9</number>
-       </property>
-       <property name="bottomMargin" >
-        <number>9</number>
-       </property>
-       <property name="horizontalSpacing" >
-        <number>6</number>
-       </property>
-       <property name="verticalSpacing" >
-        <number>6</number>
-       </property>
        <item row="0" column="0" >
         <widget class="QTextEdit" name="licenseTextEdit" >
          <property name="readOnly" >
--- qmmp-0.2.3/src/ui/forms/addurldialog.ui~	2008-07-22 13:10:39.000000000 +0200
+++ qmmp-0.2.3/src/ui/forms/addurldialog.ui	2009-01-08 16:25:59.000000000 +0100
@@ -13,18 +13,6 @@
    <string>Enter URL to add</string>
   </property>
   <layout class="QGridLayout" >
-   <property name="leftMargin" >
-    <number>5</number>
-   </property>
-   <property name="topMargin" >
-    <number>5</number>
-   </property>
-   <property name="rightMargin" >
-    <number>5</number>
-   </property>
-   <property name="bottomMargin" >
-    <number>5</number>
-   </property>
    <item row="0" column="0" colspan="3" >
     <widget class="QComboBox" name="urlComboBox" >
      <property name="editable" >
--- qmmp-0.2.3/src/ui/forms/configdialog.ui~	2008-12-02 20:53:08.000000000 +0100
+++ qmmp-0.2.3/src/ui/forms/configdialog.ui	2009-01-08 16:37:00.000000000 +0100
@@ -143,18 +143,6 @@
          <property name="spacing" >
           <number>6</number>
          </property>
-         <property name="leftMargin" >
-          <number>9</number>
-         </property>
-         <property name="topMargin" >
-          <number>5</number>
-         </property>
-         <property name="rightMargin" >
-          <number>0</number>
-         </property>
-         <property name="bottomMargin" >
-          <number>0</number>
-         </property>
          <item>
           <widget class="QGroupBox" name="groupBox" >
            <property name="title" >
@@ -280,15 +268,6 @@
        </widget>
        <widget class="QWidget" name="page" >
         <layout class="QVBoxLayout" >
-         <property name="topMargin" >
-          <number>5</number>
-         </property>
-         <property name="rightMargin" >
-          <number>0</number>
-         </property>
-         <property name="bottomMargin" >
-          <number>0</number>
-         </property>
          <item>
           <widget class="QGroupBox" name="groupBox_3" >
            <property name="title" >
@@ -378,18 +357,6 @@
        </widget>
        <widget class="QWidget" name="widget" >
         <layout class="QGridLayout" >
-         <property name="leftMargin" >
-          <number>9</number>
-         </property>
-         <property name="topMargin" >
-          <number>5</number>
-         </property>
-         <property name="rightMargin" >
-          <number>0</number>
-         </property>
-         <property name="bottomMargin" >
-          <number>0</number>
-         </property>
          <property name="spacing" >
           <number>6</number>
          </property>
@@ -537,18 +504,6 @@
        </widget>
        <widget class="QWidget" name="page_2" >
         <layout class="QVBoxLayout" >
-         <property name="leftMargin" >
-          <number>9</number>
-         </property>
-         <property name="topMargin" >
-          <number>5</number>
-         </property>
-         <property name="rightMargin" >
-          <number>0</number>
-         </property>
-         <property name="bottomMargin" >
-          <number>0</number>
-         </property>
          <item>
           <widget class="QGroupBox" name="groupBox_6" >
            <property name="title" >
@@ -637,18 +592,6 @@
        </widget>
        <widget class="QWidget" name="network" >
         <layout class="QVBoxLayout" >
-         <property name="leftMargin" >
-          <number>9</number>
-         </property>
-         <property name="topMargin" >
-          <number>5</number>
-         </property>
-         <property name="rightMargin" >
-          <number>0</number>
-         </property>
-         <property name="bottomMargin" >
-          <number>0</number>
-         </property>
          <item>
           <widget class="QGroupBox" name="groupBox_5" >
            <property name="title" >
--- qmmp-0.2.3/src/plugins/Input/mad/settingsdialog.ui~	2008-07-22 13:07:21.000000000 +0200
+++ qmmp-0.2.3/src/plugins/Input/mad/settingsdialog.ui	2009-01-19 17:40:48.000000000 +0100
@@ -174,16 +174,7 @@
         <property name="spacing" >
          <number>6</number>
         </property>
-        <property name="leftMargin" >
-         <number>0</number>
-        </property>
-        <property name="topMargin" >
-         <number>0</number>
-        </property>
-        <property name="rightMargin" >
-         <number>0</number>
-        </property>
-        <property name="bottomMargin" >
+        <property name="margin" >
          <number>0</number>
         </property>
         <item>
@@ -206,16 +197,7 @@
         <property name="spacing" >
          <number>6</number>
         </property>
-        <property name="leftMargin" >
-         <number>0</number>
-        </property>
-        <property name="topMargin" >
-         <number>0</number>
-        </property>
-        <property name="rightMargin" >
-         <number>0</number>
-        </property>
-        <property name="bottomMargin" >
+        <property name="margin" >
          <number>0</number>
         </property>
         <item>
@@ -241,16 +223,7 @@
      <property name="spacing" >
       <number>6</number>
      </property>
-     <property name="leftMargin" >
-      <number>0</number>
-     </property>
-     <property name="topMargin" >
-      <number>0</number>
-     </property>
-     <property name="rightMargin" >
-      <number>0</number>
-     </property>
-     <property name="bottomMargin" >
+     <property name="margin" >
       <number>0</number>
      </property>
      <item>
--- qmmp-0.2.3/src/plugins/Input/ffmpeg/detailsdialog.ui~	2008-07-22 13:07:26.000000000 +0200
+++ qmmp-0.2.3/src/plugins/Input/ffmpeg/detailsdialog.ui	2009-01-20 12:06:15.000000000 +0100
@@ -42,22 +42,10 @@
       <string>ASF Info</string>
      </property>
      <layout class="QGridLayout" >
-      <property name="leftMargin" >
+      <property name="margin" >
        <number>9</number>
       </property>
-      <property name="topMargin" >
-       <number>9</number>
-      </property>
-      <property name="rightMargin" >
-       <number>9</number>
-      </property>
-      <property name="bottomMargin" >
-       <number>9</number>
-      </property>
-      <property name="horizontalSpacing" >
-       <number>6</number>
-      </property>
-      <property name="verticalSpacing" >
+      <property name="spacing" >
        <number>6</number>
       </property>
       <item row="4" column="1" >
@@ -182,22 +170,10 @@
       <string>WMA Tag</string>
      </property>
      <layout class="QGridLayout" >
-      <property name="leftMargin" >
+      <property name="margin" >
        <number>8</number>
       </property>
-      <property name="topMargin" >
-       <number>8</number>
-      </property>
-      <property name="rightMargin" >
-       <number>8</number>
-      </property>
-      <property name="bottomMargin" >
-       <number>8</number>
-      </property>
-      <property name="horizontalSpacing" >
-       <number>6</number>
-      </property>
-      <property name="verticalSpacing" >
+      <property name="spacing" >
        <number>6</number>
       </property>
       <item row="6" column="1" >
