--- qmmp-0.5.3/src/plugins/Transports/mms/mmsstreamreader.cpp~	2012-01-19 10:36:29.000000000 +0100
+++ qmmp-0.5.3/src/plugins/Transports/mms/mmsstreamreader.cpp	2012-01-23 19:17:09.985110302 +0100
@@ -184,7 +184,7 @@
         m_mutex.unlock();
         if(m_aborted)
             break;
-        usleep(5000);
+        DownloadThread::usleep(5000);
     }
 }
 
--- qmmp-0.5.3/src/plugins/Transports/mms/mmsstreamreader.h~	2012-01-19 10:37:27.000000000 +0100
+++ qmmp-0.5.3/src/plugins/Transports/mms/mmsstreamreader.h	2012-01-23 19:18:55.000000000 +0100
@@ -89,6 +89,10 @@
 public:
     DownloadThread(MMSStreamReader *parent);
     virtual ~DownloadThread ();
+    static void usleep(unsigned long usecs)
+        {
+            QThread::usleep(usecs);
+        }
 
 private:
     virtual void run();
